openpype/hosts/houdini/plugins/load/load_fbx.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+"""Fbx Loader for houdini. """
+from openpype.pipeline import (
+    load,
+    get_representation_path,
+)
+from openpype.hosts.houdini.api import pipeline
+
+
+class FbxLoader(load.LoaderPlugin):
+    """Load fbx files. """
+
+    label = "Load FBX"
+    icon = "code-fork"
+    color = "orange"
+
+    order = -10
+
+    families = ["staticMesh", "fbx"]
+    representations = ["fbx"]
+
+    def load(self, context, name=None, namespace=None, data=None):
+
+        # get file path
+        file_path = self.get_file_path(context=context)
+
+        # get necessary data
+        namespace, node_name = self.get_node_name(context, name, namespace)
+
+        # create load tree
+        nodes = self.create_load_node_tree(file_path, node_name, name)
+
+        self[:] = nodes
+
+        # Call containerise function which does some automations for you
+        #  like moving created nodes to the AVALON_CONTAINERS subnetwork
+        containerised_nodes = pipeline.containerise(
+            node_name,
+            namespace,
+            nodes,
+            context,
+            self.__class__.__name__,
+            suffix="",
+        )
+
+        return containerised_nodes
+
+    def update(self, container, representation):
+
+        node = container["node"]
+        try:
+            file_node = next(
+                n for n in node.children() if n.type().name() == "file"
+            )
+        except StopIteration:
+            self.log.error("Could not find node of type `file`")
+            return
+
+        # Update the file path
+        file_path = self.get_file_path(representation=representation)
```

Comment:
Just wondering - does this work?
```suggestion
        file_path = self.get_file_path(context=representation["context"])
```

Reply:
this part was confusing because I found many things when searching the code:
This is also possible I think 
```
context = get_representation_context(representation)
```

https://github.com/ynput/OpenPype/blob/c08190ccf536864e3b94b48636637fdebc114db0/openpype/pipeline/load/utils.py#L554-L556

and in this function context is re-retrieved from representation
`context=representation["context"]`

https://github.com/ynput/OpenPype/blob/c08190ccf536864e3b94b48636637fdebc114db0/openpype/pipeline/load/utils.py#L613

---
So I chose the safest way :"D

Reply:
@iLLiCiTiT Could you tell me what do you think ?
are there any unnecessary logic in `self.get_file_path` in `load_fbx.py` ?